com.github.jeremypw.gnonograms.json

```diff
@@ -0,0 +1,46 @@
+{
+    "app-id": "com.github.jeremypw.gnonograms",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno-19.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "com.github.jeremypw.gnonograms",
+
+    "finish-args": [
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--device=dri",
+      "--filesystem=~/.local/share/com.github.jeremypw.gnonograms:create",
+      "--persist=~/.local/share/com.github.jeremypw.gnonograms"
```

Comment:
```suggestion
```
You have access to those by default. The path is `.var/app/$app-id/` instead for Flatpak apps.

Reply:
What I want is for the user to know (easily) where the game files can be found when not using Gnonograms.  Say they have received a game file from someone else and want to load it into Gnonograms.  They have to put in a location to which Gnonograms flatpak has access. So the location that appears in the FileChooserNative within the flatpak has to be the same as the one when using the system file manager.

Reply:
If, by using FileChooserNative, I can forget about filesystem access settings in the manifest, that would be an advantage.